include/seccomp.h.in

```diff
@@ -65,6 +65,7 @@ enum scmp_filter_attr {
 	SCMP_FLTATR_CTL_TSYNC = 4,	/**< sync threads on filter load */
 	SCMP_FLTATR_API_TSKIP = 5,	/**< allow rules with a -1 syscall */
 	SCMP_GLBATR_CTL_KCHECKACTS = 6,	/**< ignore kernel support checks */
```

Comment:
Ohh, GitHub is being sneaky/clever/stupid here ... I made the comment above in another commit and it is appearing here (presumably because this is the commit that introduces the code).